data/xml/2003.mtsummit.xml

```diff
@@ -0,0 +1,857 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2003.mtsummit">
+  <volume id="plenaries" ingest-date="2021-07-23">
+    <meta>
+      <booktitle>Proceedings of Machine Translation Summit IX: Plenaries</booktitle>
+      <address>New Orleans, USA</address>
+      <month>September 23-27</month>
+      <year>2003</year>
+    </meta>
+    <frontmatter>
+      <url hash="e349b78d">2003.mtsummit-plenaries.0</url>
+    </frontmatter>
```

Comment:
What is in this PDF? You could upload the PDFs so the links would work.

Related, I think: where are the "Welcome" and "Preface" from Hovy and Macklovitch?

Reply:
Strange, I have already uploaded the PDFs (to aclanthology.org). I have combined the "Welcome" and "Preface" into a single PDF and added this as frontmatter, what do you think?

Reply:
They must have got deleted with the syncing bug. Can you re-upload?

Reply:
Everything is re-uploaded now.